spec/factories/clearance.rb

```diff
@@ -4,9 +4,31 @@
   end
 
   factory :user do
-    first_name "Tony"
-    last_name "Stark"
+    first_name "Steve"
+    last_name "Rogers"
     email
     password "password"
+    role "stockholder"
+
+    factory :ironman do
+      first_name "Tony"
+      last_name "Stark"
+    end
+  end
+
+  factory :admin, class: User do
+    first_name "Admin"
+    last_name  "User"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.